robot/Cumulus/resources/LevelsPageObject.py

```diff
@@ -46,11 +46,9 @@ def _is_current_page(self):
 
     def go_to_edit_level_page(self,levelid=None):
         """ Navigates to the edit view for the given level id """
-        url = self.cumulusci.org.lightning_base_url
         name = "Level_c"
```

Comment:
can you please rename this to `objectname` as well, as I see you are using `objectname` in other two files?

Reply:
done